SMA_FluxExtraction.ipynb

```diff
@@ -0,0 +1,634 @@
+{
```

Comment:
<p>The multipication of line 1 by u.Hz can lead to an issue where the frequency is off by a factor of 10^9. Should this be changed to multiplication by u.GHz?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Smithsonian/sma-science-tutorials/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>